.github/workflows/tests.yml

```diff
@@ -91,3 +91,14 @@ jobs:
 
       - name: Send coverage to codecov.io
         run: bash <(curl -s https://codecov.io/bash)
+  shellcheck:
+    name: "🐚 Shellcheck"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run ShellCheck
+      uses: ludeeus/action-shellcheck@master
```

Comment:
Maybe pin this down to a specific tag?

Reply:
Yes, that is a good idea. Thanks :). I'll use the same as you did in osbuild-koji.